packages/api/cms-api/src/user-permissions/user-permissions.service.ts

```diff
@@ -195,4 +198,23 @@ export class UserPermissionsService {
                 return p;
             });
     }
+
+    async hasEqualOrMorePermissions(user: User | CurrentUser, targetUser: User): Promise<boolean> {
```

Comment:
would be easy to unit test this (it is security relevant)

Reply:
Yes. Please add unit tests for this